pkgs/development/libraries/science/astronomy/indilib/indi-3rdparty.nix

```
@@ -32,23 +35,23 @@ stdenv.mkDerivation rec {
 
   buildInputs = [
     indilib libnova curl cfitsio libusb1 zlib boost gsl gpsd
-    libjpeg libgphoto2 libraw libftdi1 libdc1394 ffmpeg fftw
+    libjpeg libgphoto2 libraw.dev libraw.lib libftdi1 libdc1394 ffmpeg fftw
```

I think there is no change necessary here. If I understand `mkDerivation` correctly, then for each item in `buildInputs`, the `dev` output is used automatically (if present) and `libraw.dev` already includes `libraw.lib` (it has a propagated build input to `libraw.lib`).

```suggestion
    libjpeg libgphoto2 libraw libftdi1 libdc1394 ffmpeg fftw
```

---

Ah that was a leftover from my experiments to get it working :grimacing: